### ISSUE TITLE
Cannot disable accessibility on Touchables

### ISSUE THREAD

**Issue by Gguigre** [facebook/react-native#27683]
This issue is directly linked to https://github.com/facebook/react-native/issues/25921 which is closed for being stale but not resolved.

Setting props `accessible={false} accessibilityElementsHidden importantForAccessibility="no-hide-descendants"` on **touchables** does not prevent the component from being seen as an accessible group.

I've tested with both <TouchableOpacity /> and <TouchableWithoutFeedback />

React Native version:
```
System:
    OS: macOS Mojave 10.14.6
    CPU: (8) x64 Intel(R) Core(TM) i5-8259U CPU @ 2.30GHz
    Memory: 1.11 GB / 16.00 GB
    Shell: 5.3 - /bin/zsh
  Binaries:
    Node: 12.6.0 - /var/folders/b_/6x4_gw1x66b57b1sfhm9m50m0000gn/T/yarn--1578306196357-0.06139298171646712/node
    Yarn: 1.21.1 - /var/folders/b_/6x4_gw1x66b57b1sfhm9m50m0000gn/T/yarn--1578306196357-0.06139298171646712/yarn
    npm: 6.9.0 - ~/.nvm/versions/node/v12.6.0/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.2, DriverKit 19.0, macOS 10.15, tvOS 13.2, watchOS 6.1
    Android SDK:
      API Levels: 23, 24, 25, 26, 27, 28
      Build Tools: 23.0.1, 26.0.2, 26.0.3, 27.0.3, 28.0.1, 28.0.3, 29.0.2
  IDEs:
    Android Studio: 3.1 AI-173.4819257
    Xcode: 11.2.1/11B500 - /usr/bin/xcodebuild
  npmGlobalPackages:
    react-native-cli: 2.0.1
    react-native-plugin-make: 1.0.1
```
## Steps To Reproduce

1. Add a touchable component to the view with the accessible property set explicitly to false: accessible={false}. Try adding the `importantForAccessibility={'no-hide-descendants'}` (Android) and `accessibilityElementsHidden` (iOS) property as well and the bug still occurs.
2. Open the app on a device. Enable TalkBack/VoiceOver. Use the swipe gesture to navigate through accessible components.

**Expected behaviour:**
I would expect the elements with those props to be non selectable by the screenReader

**Snack, code example, screenshot, or link to a repository:**
https://snack.expo.io/@guigre/5aee23

**Comment by Gguigre** [comment]
Fixed in next release-candidate (0.62)
